components/update_user_group_modal/index.ts

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import {connect} from 'react-redux';
+import {ActionCreatorsMapObject, bindActionCreators, Dispatch} from 'redux';
+
+import {Action, ActionResult} from 'mattermost-redux/types/actions';
+
+import {GlobalState} from 'types/store';
+
+import {CustomGroupPatch} from 'mattermost-redux/types/groups';
+import {patchGroup} from 'mattermost-redux/actions/groups';
+import {ModalData} from 'types/actions';
+import {openModal} from 'actions/views/modals';
+import {getGroup} from 'mattermost-redux/selectors/entities/groups';
+
+import UpdateUserGroupModal from './update_user_group_modal';
+
+type OwnProps = {
+    groupId: string;
+}
+
+function makeMapStateToProps() {
```

Comment:
Again, we don't need here to use `makeMapStateToProps`, we can just use `mapStateToProps`